R/github.R

```diff
@@ -67,3 +67,15 @@ github_pat <- function() {
   message("Using github PAT from envvar GITHUB_PAT")
   pat
 }
+
+github_DESCRIPTION <- function(username, repo, ref = "master") {
+  response <- github_GET(
+    file.path("repos", username, repo, "contents", "DESCRIPTION"),
+    httr::write_memory())
+
+  content <- rawToChar(base64enc::base64decode(response$content))
```

Comment:
The github API returns a JSON object with a content field that is base64 encoded.  I think httr will decode automatically if the entire response is base64 encoded, but only part of it is in this case.

If there is a httr function I can use to do this decoding that would be great, I just missed it!